testpilot/frontend/static-src/app/views/footer-view.js

```diff
@@ -1,12 +1,32 @@
+import app from 'ampersand-app';
 import BaseView from './base-view';
 
 export default BaseView.extend({
   template: `<div id="footer-links">
+                <div class="legal-links">
```

Comment:
![screen shot 2016-08-18 at 4 20 08 pm](https://cloud.githubusercontent.com/assets/3323249/17789320/bdef9e70-655f-11e6-89a1-b2514a45111a.png)

@pdehaan wontfix on this b/c like 5 is the one that's off!